### ISSUE TITLE
[Proposal] Adding specific(like already done env) env reset function in AsyncVectorEnv

### ISSUE THREAD

**Issue by seolhokim** [openai/gym#2555]
### Proposal 

Adding specific(like already done env) env reset function in AsyncVectorEnv

### Motivation

AsyncVectorEnv is suitable for creating an environment in parallel. This can greatly shorten the time. But there is no separately reset function that is quite important to make fixed length training. 

### Pitch

```
def done(only_finished=False):
~~~
```
The environments are asynchronously reset.

### Alternatives

make env_done_lst in AsyncVectorEnv
```
for pipe, single_seed in zip(self.parent_pipes, seed):
  if (only_finished == True) and self.env_done_lst[single_seed]:
     continue
  pipe.send(("reset", single_seed))
```

### Additional context
I can create a pull-request about this.

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/openai/gym/issues) in the repo (**required**)

**Comment by RedTachyon** [comment]
What's wrong with regular `env.reset()`? It's implemented here: https://github.com/openai/gym/blob/master/gym/vector/vector_env.py#L55

**Comment by seolhokim** [comment]
When using some classes, which inherited Class AsyncVectorEnv, I put integer argument into the reset function, there was a positional argument error so I thought it was not implemented. I will check again. Thanks
